user-journey-tests/page-objects/destination/destinationAnswersPage.js

```diff
@@ -1,7 +1,21 @@
 import { Page } from '../page.js'
 
+const pageHeadingAndTitle =
+  'Check your answers before you continue your application'
+
 class DestinationAnswersPage extends Page {
   pagePath = 'destination/check-answers'
+  pageHeading = pageHeadingAndTitle
+  pageTitle = pageHeadingAndTitle
+
+  // Change links
+  get changeDestinationLink() {
+    return $('[data-testid="destinationType-change-link"]')
+  }
+
+  get destiationValue() {
+    return $$('.govuk-summary-list__value')[0]
```

Comment:
prefer `.at(0)`

Reply:
Just tried this and didn't work, caused failures